src/java.base/share/classes/java/util/Digits.java

```diff
@@ -33,7 +33,7 @@
  *
  * @since 21
  */
-sealed interface Digits permits DecimalDigits, HexDigits, OctalDigits {
+sealed interface Digits permits HexDigits, OctalDigits {
```

Comment:
Don't break up the trio, move all three classes and the interface to jdk.internal.util.
I don't see the value in the INSTANCE values but keep it intact.

Reply:
HexDigits I will change after the PR #1745 is merged,  Can you add comment /sponsor for me?

Reply:
I want to remove the Digits interface and use static methods

Reply:
I agree with @RogerRiggs that these should be moved together and with as few changes as possible. We can do redesigns in follow-ups.

I'd be OK with adding static variants of each method as needed, leaving the instance methods unchanged. 

I'll note that these `java.util.Digits` came in as part of what's currently a preview feature (https://openjdk.org/jeps/430) and the author (@JimLaskey) might have plans that requires an implementation that could be passed around in the final version - so design changes should be coordinated with him.